ramp4-pcar4/storylines-editor
# add ability to change product UUID

### Related Item(s)
#253 

### Changes
- Adds ability to change the name of existing Storylines products.

### Notes
The old UUID does not automatically get deleted at the moment. We'll need something on the server side to clean this up.

### Testing
Steps:
1. Open the editor and load an existing product.
2. Try to change the name of the product and save it. Ensure the config files are updated correctly and that everything still works.
3. Try to change the name of the product to a UUID that already exists. Ensure that the warning pops up.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/storylines-editor/288)
